loki-monitoring/dashboards/loki-monitoring-dashboard.json

```diff
@@ -422,7 +422,7 @@
             },
             "disableTextWrap": false,
             "editorMode": "builder",
-            "expr": "sum by(app_kubernetes_io_part_of) (kube_deployment_status_replicas_unavailable{namespace=\"{{ .Values.loki.namespace }}\"} > 0)",
```

Comment:
Why do we make this change?

Reply:
Original one showed `kube-metrics`, the component that contributes to scraping. `deployment` represents the name of the failed deployment. This change is fixing a bug.